### PR TITLE
Add tx manager http docs

### PR DESCRIPTION
**Pull request by SatpalSandhu61**
Documentation for the following code change: https://github.com/ConsenSys/quorum/pull/1109

### CODE REVIEW

File: docs/Concepts/Privacy/PrivateTransactionManager.md [hunk 1]
```diff
@@ -11,52 +11,158 @@ The Private Transaction Manager has two distinct components:
...
-node.
+Enable private transactions using any of the following methods when starting a GoQuorum node:
 
-Use this `PRIVATE_CONFIG` environment variable either with a direct path to the Privacy Manager's `.ipc` socket
-or with a path to a TOML IPC configuration file as explained in the following sections.
+* Set the `PRIVATE_CONFIG` environment variable to specify a direct IPC connection to the private transaction manager.
+* Set the `PRIVATE_CONFIG` environment variable to a TOML configuration file that specifies the private transaction manager connection.
+* Use command line parameters to specify the private transaction manager connection.
+
+These options will be explained in detail in the sections below.
```
Comment (NicolasMassart) [inline docs/Concepts/Privacy/PrivateTransactionManager.md:20]:
```suggestion
These options will be explained in detail in the following sections.
```

File: docs/Concepts/Privacy/PrivateTransactionManager.md [hunk 2]
```diff
@@ -11,52 +11,158 @@ The Private Transaction Manager has two distinct components:
...
+    !!! example "Example IPC connection"
+
+        ```bash
+        geth <other parameters> \
+             --ptm.socket qdata/c1/tm.ipc
+        ```
+
+=== "HTTP connection"
+
+    This should only be used for development purposes, due to a lack of security on the connection.
```
Comment (NicolasMassart) [inline docs/Concepts/Privacy/PrivateTransactionManager.md:143]: maybe tu this warning in a `!!! warning` box

File: docs/Concepts/Privacy/PrivateTransactionManager.md [hunk 3]
```diff
@@ -11,52 +11,158 @@ The Private Transaction Manager has two distinct components:
...
+        * `tlsClientKey` is file containing client certificate private key
+
+## Using command line parameters
+
+The command line parameters listed below can be used to configure the connection to the Transaction Manager.
+Note that these can be used in conjunction with the other options given above,
+in which case the command line parameters will override any others.
+
+```text
+QUORUM PRIVATE TRANSACTION MANAGER OPTIONS:
```
Comment (NicolasMassart) [inline docs/Concepts/Privacy/PrivateTransactionManager.md:115]: A proper markdown formatting would make this more readable. I guess it's a copy of the CLI help output, but if it doesn't change so often, the benefit of having a nice format against having to update by spotting changes is in favor of nice format IMO.

File: docs/Concepts/Privacy/PrivateTransactionManager.md (hunk 1 above)
Comment (SatpalSandhu61) [reply docs/Concepts/Privacy/PrivateTransactionManager.md]: done

File: docs/Concepts/Privacy/PrivateTransactionManager.md (hunk 2 above)
Comment (SatpalSandhu61) [reply docs/Concepts/Privacy/PrivateTransactionManager.md:132]: done

File: docs/Concepts/Privacy/PrivateTransactionManager.md (hunk 3 above)
Comment (SatpalSandhu61) [reply docs/Concepts/Privacy/PrivateTransactionManager.md]: Amended. Let me know if this format looks ok.

**Review by NicolasMassart — APPROVED** [review]
thanks!

**Pull request merged**